### ISSUE TITLE
Merged PRs on Release notes on Github are wrong

### ISSUE THREAD

**Issue by laborg** [JuliaDSP/DSP.jl#568]
Looking at https://github.com/JuliaDSP/DSP.jl/releases for the latest 0.7.10 it appears that the list of merged PRs is wrong. Example: It's stated that for 0.7.10 the PR #458 was merged, which made me nervous because it would be breaking for our code base. But apparently the PR is not part of the 0.7.10 release on the code level...

**Comment by martinholters** [comment]
Good catch. That's the list for 0.8.0 once it lands. I'll update...

**Comment by martinholters** [comment]
This is the old version which I'm preserving here as I imagine those entries will be missing from what the tagbot will generate for 0.8.0.

---

## DSP v0.7.10

[Diff since v0.7.9](https://github.com/JuliaDSP/DSP.jl/compare/v0.7.9...v0.7.10)


**Merged pull requests:**
- Add `hann` export to be equal to `hanning` (#448) (@staticfloat)
- Enable Analog Filter Design (#458) (@gruberchr)
- Use length of arrays directly in conv (#494) (@jishnub)
- Update PolynomialRatio docs. (#495) (@kcin96)
- Support `dims` kwarg in `rms` (#500) (@martinholters)
- Create WelchConfig object (#502) (@haberdashPI)
- Add frequency estimators by Jacobsen and Quinn (#503) (@mbaz)
- CompatHelper: add new compat entry for Statistics at version 1, (keep existing compat) (#510) (@github-actions[bot])
- replace splat with `Tuple` (#513) (@wheeheee)
- update tukey docs (#514) (@haakon-e)
- `filt` perf optimizations (#516) (@wheeheee)
- extend `lpc` to complex inputs, other fixes (#517) (@wheeheee)
- add Zenodo badge to README.md (#519) (@maanikmarathe)
- Add doctest to Xcorr (#520) (@ErikBuer)
- Bump Documenter.jl to v1, fix typos (#526) (@wheeheee)
- Adding documentation for bilinear function (#527) (@anchal-physics)
- Delete NEWS.md (#529) (@ViralBShah)
- Update LICENSE.md (#530) (@ViralBShah)
- Bump the minimum required Julia version to 1.6 (#532) (@martinholters)
- add type as another possible argument to `DF2TFilter` (#534) (@wheeheee)
- Update Project.toml (#535) (@ViralBShah)
- general cleanups (#536) (@wheeheee)
- Use Bessels.jl for Bessel function calculations (#538) (@anowacki)
- Fix bounds errors when resampling with some arbitrary ratios (#539) (@anowacki)
- use `std` to test lpc error (#542) (@wheeheee)
- Add Apple silicon to CI (#544) (@ViralBShah)
- Fix docstring typo in `DF2TFilter` (#550) (@scls19fr)
- Reduce CI load (#551) (@ViralBShah)
- Annotate docstring arguments with types in Filters (#552) (@wheeheee)
- Don't fail if codecov commit fails (#553) (@ViralBShah)
- Update ESPRIT documentation. (#554) (@kcin96)
- Fix `welch_pgram!`, update periodogram docs (#557) (@kcin96)
- estimation: Fix line breaks in Quinn/Jacobsen references (#558) (@dnadlinger)
- [DOCS] Add Small Note about Method Validations (#561) (@TheCedarPrince)
- Update CI.yml to use pre-release builds (#564) (@ViralBShah)
- [release-0.7] Update CI.yml to use pre-release builds (#565) (@martinholters)
- Backport `filt` fix (#516 and #539) and bump version to 0.7.10 (#566) (@martinholters)

**Closed issues:**
- single-chunk and multi-chunk runs with streaming arbitrary resampling don't match (#134)
- ControlSystems + DSP (#145)
- export reset! (#146)
- welch_pgram index out of bounds (#155)
- Resample segmentation fault for rate::AbstractFloat (#262)
- Citability (#311)
- Got "BoundsError" while trying to use `resample` function. (#317)
- Performance regression in conv for images (#339)
- analogfilter design seems to be broken (#341)
- Setting up 2FA for JuliaDSP (#363)
- InexactError with stateful filter and complex data (#371)
- DSP.jl still exports fftfreq (#406)
- Order of polynomial coefficients (#415)
- [DOC] Inaccuracy in the Esprit Algorithm Documentation Code (#416)
- Power, or power spectral density of Periodogram? (#480)
- Make `rms` work on dimension of arrays (#499)
- `kaiser` performance: use Bessels.jl? (#506)
- No documentation for DSP.Filters.bilinear() (#518)
- Add inbounds to methods in design.jl  (#540)
- Doc issues on JuliaHub with 0.7.9 - Missing docstring. Check Documenter's build log for details. (#547)
- Docstring error - f parameter (#549)
- Documentation (#560)
- Maybe filter example need to be updated? (#567)
